pkg/plugins/runner/runner_test.go

```diff
@@ -0,0 +1,34 @@
+package runner
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/nornir-automation/gornir/pkg/gornir"
+)
+
+var (
+	testHosts = map[string]*gornir.Host{
+		"dev1": {Hostname: "dev1"},
+		"dev2": {Hostname: "dev2"},
+		"dev3": {Hostname: "dev3"},
+		"dev4": {Hostname: "dev4"},
+	}
+)
```

Comment:
I would prefer to avoid global variables. I think is better you add this to the test function itself.

Reply:
Ok, I usually don't care much about globals in tests but will change. Don't expect a lot of duplication here